content/login-from-live-disk.md

```diff
@@ -19,7 +19,7 @@ section: software-troubleshooting
 tableOfContents: true
 ---
 
-## Login from Live Disk (Chroot)
+## Log in from Live Disk (Chroot)
```

Comment:
If we're changing "Login" to "Log in" here, we should also change it in the article title for consistency's sake.